syntax = "proto3";

package ce;

// ====== CeService =======
service CeService {
	rpc SendInfo (Info) returns (Response) {}
	rpc QueryInfo (Params) returns (Response) {}
}

message Response {
	int32 status = 1;
	string content = 2;
}

// SendInfo
message Info {
	string info = 1;
}

// QueryInfo
message Params {
	string params = 1;
}
